packages/opentelemetry-types/src/metrics/Handle.ts

```diff
@@ -14,10 +14,32 @@
  * limitations under the License.
  */
 
-export interface CounterTimeseries {
-  // Adds the given value to the current value. Values cannot be negative.
+/** A Handle for Counter Metric. */
+export interface CounterHandle {
```

Comment:
Rename to counter to cumulative

Reply:
Btw, I thought we have decided to go with `Counter` instead of `Cumulative`

Reply:
cool beans just saw the PR in the rfc section to change it back to counter